packages/unity-js-bridge/src/thirdweb-bridge.ts

```diff
@@ -143,7 +143,7 @@ class ThirdwebBridge implements TWBridge {
       }
       (globalThis as any).X_SDK_NAME = "UnitySDK_WebGL";
       (globalThis as any).X_SDK_PLATFORM = "unity";
-      (globalThis as any).X_SDK_VERSION = "4.5.1";
+      (globalThis as any).X_SDK_VERSION = "4.6.0";
```

Comment:
There's no way to automate this?

Reply:
no, I make builds before every version either way to grab latest ts updates